tests/_fixtures/config/no-plugins.json

```diff
@@ -1,7 +1,10 @@
 {
   "uiConfiguration": {
+    "features": {
+      "maintenance": true
```

Comment:
I think the integration test is failing, because this has to include Quota as well.

Reply:
oh boy -.-
copy-paste...

Reply:
It should be fine now, I rebased.